GenderPayGap.WebJob/Functions/FrequentTimerTriggeredWebJobs/Functions.Search.cs

```diff
@@ -64,48 +64,35 @@ public async Task UpdateSearchAsync(ILogger log, string userEmail = null, bool f
             string userEmail = null,
             bool force = false)
         {
-            if (RunningJobs.Contains(nameof(UpdateSearchAsync)))
+            await searchRepositoryToUpdate.CreateIndexIfNotExistsAsync(indexNameToUpdate);
+
+            if (typeof(T) == typeof(EmployerSearchModel))
             {
-                log.LogInformation("The set of running jobs already contains 'UpdateSearch'");
-                return;
+                await AddDataToIndexAsync(log);
             }
-
-            try
+            else if (typeof(T) == typeof(SicCodeSearchModel))
+            {
+                await AddDataToSicCodesIndexAsync(log);
+            }
+            else
             {
-                await searchRepositoryToUpdate.CreateIndexIfNotExistsAsync(indexNameToUpdate);
+                throw new ArgumentException($"Type {typeof(T)} is not a valid type.");
+            }
 
-                if (typeof(T) == typeof(EmployerSearchModel))
-                {
-                    await AddDataToIndexAsync(log);
-                }
-                else if (typeof(T) == typeof(SicCodeSearchModel))
-                {
-                    await AddDataToSicCodesIndexAsync(log);
-                }
-                else
+            if (force && !string.IsNullOrWhiteSpace(userEmail))
+            {
+                try
                 {
-                    throw new ArgumentException($"Type {typeof(T)} is not a valid type.");
+                    await _Messenger.SendMessageAsync(
+                        "UpdateSearchIndexes complete",
+                        userEmail,
+                        "The update of the search indexes completed successfully.");
                 }
-
-                if (force && !string.IsNullOrWhiteSpace(userEmail))
+                catch (Exception ex)
                 {
-                    try
-                    {
-                        await _Messenger.SendMessageAsync(
-                            "UpdateSearchIndexes complete",
-                            userEmail,
-                            "The update of the search indexes completed successfully.");
-                    }
-                    catch (Exception ex)
-                    {
-                        log.LogError(ex, "UpdateSearch: An error occurred trying to send an email");
-                    }
+                    log.LogError(ex, "UpdateSearch: An error occurred trying to send an email");

```

Comment:
Should we be using the `CustomLogger` here instead?

Reply:
Yes. I'll add a ticket to replace all "old-style" logging with CustomLogger